cloudferrylib/os/actions/set_volume_id.py

```diff
@@ -0,0 +1,15 @@
+from cloudferrylib.base.action import action
+import copy
+
+
+class SetVolumeId(action.Action):
+
```

Comment:
Could you please add some description to this action? It would be really helpful.